ppmpy/ppm.py

```diff
@@ -12148,6 +12151,80 @@ def sphericalHarmonics_lmax(self, radius):
 
         return lmax, N, npoints
 
+    def get_power_spectrum(self, varloc, dump_start, dump_stop, dump_step=1, radius=None, mass=None):
+        '''
+        Calculates the power spectrum of variable 'varloc' at radius 'radius' for
```

Comment:
add to docstring that dump range is for averaging

format for docstring:

one line (like header), one empty line, then multi-line further explanation